.circleci/config.yml

```diff
@@ -502,56 +587,66 @@ jobs:
       - checkout
       - run: .circleci\windows\prepare.bat
 
-      - run: .circleci\windows\python\deploy_snapshot.bat
-
-      - run: .circleci\windows\java\deploy_snapshot.bat
-
-      - run: .circleci\windows\clib\deploy_snapshot.bat
-      - run: .circleci\windows\clib\test_assembly.bat
+#      - run: .circleci\windows\python\deploy_snapshot.bat
```

Comment:
Don't forget to uncomment these once we're about to merge it :)

Reply:
Sure! It's already stashed